src/proxies/clientSDKLogProxy.ts

```diff
@@ -1,47 +1,26 @@
 import { CogniteClient } from '@cognite/sdk';
 import { version } from '../../package.json';
-import { ClientSDKContextType } from './clientSDKContext';
-import { ClientSDKProxyContextType } from './clientSDKProxyContext';
+import { ClientSDKContextType } from '../context/clientSDKContext';
 
-const apiNames = [
-  'assets',
-  'timeseries',
-  'datapoints',
-  'sequences',
-  'events',
-  'files',
-  'raw',
-  'projects',
-  'groups',
-  'securityCategories',
-  'serviceAccounts',
-  'models3D',
-  'revisions3D',
-  'files3D',
-  'assetMappings3D',
-  'viewer3D',
-  'apiKeys',
-  'login',
-  'logout',
-] as const;
-
-type ClientApiName = typeof apiNames[number];
+// Exclude 'project' cause it returns string from SDK instance and doesn't make
+// any request to API
+type ClientApiName = Exclude<keyof CogniteClient, 'project'>;
 type ClientApi = CogniteClient[ClientApiName];
 
 const GearboxHeader = `CogniteGearbox:${version}`;
+const excludeMethods = ['setOneTimeSdkHeader', 'project'];
```

Comment:
what about `loginWithOAuth`, `loginWithApiKey`, `setBaseUrl`, `getMetadata`, `get, put, post, delete`?

Reply:
Method names in array is names that we want to exclude from tracking for sure. We can always add other methods here

Reply:
but we definitely want to exclude those above for sure, is there a reason why we are not adding them rn?

Reply:
no, actually, so I'll add them